flow360/component/results/case_results.py

```diff
@@ -42,6 +43,53 @@ def _temp_file_generator(suffix: str = ""):
     return file_path
 
 
+def _find_by_pattern(all_entities: list, pattern):
+
+    matched_entities = []
```

Comment:
Can we use some other name than entities? `Entities` are usually associated with surface slices etc in context of python client.

Reply:
addressed in: https://github.com/flexcompute/Flow360/pull/561/commits/7e8543c64dc6fbf027741165109bec18ccd51a6c